### PR TITLE
Adding items schema

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: lib/repository.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+export const DB_NAME = 'helloTigris'
+export const COLLECTION_NAME = 'todoItems'
+
+// TODO: Add error handling
+export async function getAllItems (limit = 20, skip = 0): Promise<Array<TodoItem>> {
+  const db = tigris.getDatabase(DB_NAME)
+  const options = new ReadRequestOptions(limit, skip)
+  const collection: Collection<TodoItem> = db.getCollection(COLLECTION_NAME)
+  const cursor = collection.findMany(undefined, undefined, undefined,
```
Comment (ovaistariq) [inline lib/repository.ts:14]: all the `undefined`s as params look weird

File: lib/tigris.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ if (!process.env.TIGRIS_HOST) {
 
 let tigris: Tigris
 const tigrisHost: string = process.env.TIGRIS_HOST as string
```
Comment (ovaistariq) [inline lib/tigris.ts:8]: It is better to call it TIGRIS_URI.

File: scripts/setup.mjs [hunk 3]
```diff
@@ -2,53 +2,51 @@ import { TigrisDataTypes } from '@tigrisdata/core/dist/types.js'
 import { Tigris } from '@tigrisdata/core'
 import dotenv from 'dotenv'
 
-dotenv.config({ path: '.env.local'});
-const inputUrl = process.env.TIGRIS_HOST;
+// TODO: Add error handling
+dotenv.config({ path: '.env.local' })
```
Comment (ovaistariq) [inline scripts/setup.mjs:6]: Load the config file based on whether it is local or production environment

File: scripts/setup.mjs [hunk 4]
```diff
@@ -2,53 +2,51 @@ import { TigrisDataTypes } from '@tigrisdata/core/dist/types.js'
...
+const tigris = new Tigris({ serverUrl: inputUrl, insecureChannel: true })
 
 // setup db
-const db = await tigris.createDatabaseIfNotExists(DB_NAME);
+const db = await tigris.createDatabaseIfNotExists(DB_NAME)
 console.log(`Created database: ${DB_NAME}`)
 
 // schema definition
-const taskListSchema = {
+const todoItemSchema = {
```
Comment (ovaistariq) [inline scripts/setup.mjs:19]: The schema should be defined together with the model's interface definition.

Comment (adilansari) [reply scripts/setup.mjs:19]: standalone ECMA script cannot access anything in next runtime modules, hence duplicating constants, schema etc.

File: scripts/setup.mjs (hunk 3 above)
Comment (adilansari) [reply scripts/setup.mjs:6]: ack

File: lib/tigris.ts (hunk 2 above)
Comment (adilansari) [reply lib/tigris.ts:8]: ack

File: lib/repository.ts (hunk 1 above)
Comment (adilansari) [reply lib/repository.ts:14]: keeping a checklist of ts client improvements for next revision

File: scripts/setup.mjs (hunk 4 above)
Comment (ovaistariq) [reply scripts/setup.mjs:19]: If the schemas are defined in a separate directory, they can still be imported as modules?

File: lib/repository.ts (hunk 1 above)
Comment (adilansari) [reply lib/repository.ts:14]: To get rid of undefined, one option is to change signature to:

```typescript
	findMany(
		args: {
			filter?: Filter<T>,
			readFields?: ReadFields,
			tx?: Session,
			options?: ReadRequestOptions
		}
	): Cursor<T> {
	// impl
	}
```

Comment (ovaistariq) [reply lib/repository.ts:14]: It is worth exploring, the only issue is that it is a breaking change. How do you feel about that?

**Pull request merged**